### PR TITLE
Can you add build static library for code.c, jit.c, profiler.c and etc

### PR DESCRIPTION
**Pull request by malikzh**
Hi! I making binding for Go, for my purposes. Can you add target to build hl static library?

Go bindings for hashlink:https://github.com/malikzh/gohl

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -206,10 +212,13 @@ release_osx:
 .c.o :
 	${CC} ${CFLAGS} -o $@ -c $<
 
+clean_a:
+	rm -f hls.a
```
Comment (Aurel300) [inline Makefile:216]: Should be prefixed with `lib`.

**Comment by Aurel300** [comment]
Why `libhls.a` and not `libhl.a`? You can still refer to the statically-built version by using the full path.

File: Makefile (hunk 1 above)
Comment (malikzh) [reply Makefile:216]: agree

**Comment by malikzh** [comment]
> Why `libhls.a` and not `libhl.a`? You can still refer to the statically-built version by using the full path.

Fixed

**Comment by Aurel300** [comment]
Why close?

**Pull request closed without merging**